docs: edited the making a data request page to reflect the changes to… (#79)
stjudecloud/university: docs/genomics-platform/requesting-data/making-a-data-request/index.md

#### Helpful Things to Remember

1. Data in St. Jude Cloud is grouped into [Data Access Units (DAUs)](../glossary/#data-access-unit), which usually correspond to large-scale sequencing initiatives at St. Jude. 
2. Individuals can [apply for access](../glossary/#data-access-agreement) to DAUs on a case-by-case basis for a specific amount of time (1 year, with the opportunity to extend access).
3. Access to data in a given DAU is assessed by the corresponding [Data Access Committee](../glossary/#data-access-committee) who reviews a variety of factors to grant access.
4. There are a number of terms of use and restrictions outlined in the [Data Access Agreement](https://platform.stjude.cloud/api/v1/data_access_agreements/). Everyone who will be working with the data must understand and agree to these terms.

## Selecting Data

![](./data-browser-overview.png)

> The primary way to make your data selection is through our [Genomics Platform Data Browser](https://platform.stjude.cloud/requests/diseases). You can search our raw genomic data by diagnosis, publication, study, or individual samples by selecting a tab along the top. You may further refine your search by applying filters from three categories: Sequencing Type, Sample Type, and File Type. Please note that applying multiple filters within the same category filters using 'OR' logic, while applying multiple filters across different categories filters using 'AND' logic. For example in the overview image above, we have filtered the browser to only show data that is (either WGS OR WES) AND (Diagnosis OR Relapse) AND BAM.

Reviewer: `/data` is a fine path to use as well, as it will redirect to `/data/diseases/paired-tumor-normal` by default, or whatever the previous tab the user was on last if possible.

```suggestion
The primary way to make your data selection is through our [Genomics Platform Data Browser](https://platform.stjude.cloud/data/diseases/paired-tumor-normal). You can search our raw genomic data by diagnosis, publication, study, or individual samples by selecting a tab along the top. You may further refine your search by applying filters from three categories: Sequencing Type, Sample Type, and File Type. Please note that applying multiple filters within the same category filters using 'OR' logic, while applying multiple filters across different categories filters using 'AND' logic. For example in the overview image above, we have filtered the browser to only show data that is (either WGS OR WES) AND (Diagnosis OR Relapse) AND BAM.
```

Alternatively, you may be directed to the Genomics Platform Data Browser through another App to request specific samples. The [PeCan homepage](../../../pecan/#requesting-raw-genomics-through-pecan) is one such app that allows you to select data through an interactive visualization.

<!-- After clicking on *Request Data*, we ask that you review your selection and make sure that the [DAUs](../glossary/#data-access-unit) corresponding to the set of data you have selected is indeed the data you want to request. -->

<!--![](./request-data-select-DAUs.png)-->

## Selecting a Project or Creating a New One

> Now that you have selected your data, you need to choose whether to vend this data into an existing St. Jude Cloud project on DNAnexus, or to create a new St. Jude Cloud project. On the "Choose Your Data Destination" step, select to use an existing project and then which specific project you want to add data to, or select a new project.

Reviewer: I think this could use some minor clarification.
```suggestion
Now that you have selected your data, you need to choose whether to vend this data into an existing St. Jude Cloud project on DNAnexus, or to create a new St. Jude Cloud project. On the "Choose Your Data Destination" step, select to use an existing project and then which specific project you want to add data to, or select to create a new project.
```

On the "Name Your Project" step, you will see the name of the existing project you selected or a blank space to add a name for a new project. In either case, you also need to name a folder within the project. The text box will be pre-filled with "Data Requested (date)", but you are able to change it here, and also can edit any names after you have created your project.

!!!note Important Information About Vending to Existing Projects
* Within a project, if you use the name of an existing folder, the new data will be added to that folder, a duplicate named folder will not be created.
* Selected files already in the project will not be added again - there can't be duplicate files within a project, even in different folders.

> * If you delete files from your project/folder, and then those files are included in a future selection, they will once again be added.

Reviewer: 👍🏼

!!!

### Example Use Case

Let's quickly look at our Real-Time Clinical Genomics (RTCG) dataset as an example of how projects, folders, and files function together.

> The first time you request access to RTCG from the Studies Tab of the databrowser, you will name your new project, for this example we'll refer to it as **RTCG Data Project**. Then you name the folder for this initial selection of data, today's date will autofill, but instead we'll refer to the folder as **Initial Request Folder**. After that you complete the DAA and the rest of the process, and once you are approved the data is delivered within your **Initial Request Folder** folder inside of **RTCG Data Project**.

Reviewer: We sometimes use `Data browser`, `Databrowser`, and here we use `databrowser`. Should be make all references to this consistent? FWIW, I'm not clear which version is better or correct.

> The first time you request access to RTCG from the Studies Tab of the databrowser, you will name your new project, for this example we'll refer to it as **RTCG Data Project**. Then you name the folder for this initial selection of data, today's date will autofill, but instead we'll refer to the folder as **Initial Request Folder**. After that you complete the DAA and the rest of the process, and once you are approved the data is delivered within your **Initial Request Folder** folder inside of **RTCG Data Project**.

Reviewer: ```suggestion
The first time you request access to RTCG from the Studies Tab of the databrowser, you will name your new project. For this example, we'll refer to it as **RTCG Data Project**. Then you'll name the folder for this initial selection of data. Today's date will autofill, but we'll refer to the folder as **Initial Request Folder** instead. After that, complete the DAA and the rest of the process. Once you are approved, the data is delivered within your **Initial Request Folder** folder inside of **RTCG Data Project**.
```

> The RTCG dataset is continuously updated with more Clinical Genomics data from St. Jude, and upon receiving an email about the latest release to the dataset, you return to the Databrowser and once again select RTCG from the Studies Tab. Rather than create a new project, you can select to use an existing one, and then select **RTCG Data Project** from the dropdown menu. Now you name a new folder, we'll call this one **New Release Folder** and submit the request - since you already have access there is no need to complete a new DAA or collect any signatures. There will now be two folders within your **RTCG Data Project** with the **Initial Request Folder** having most of the data, while **New Release Folder** only has the new files that have been released since the first request was made, and therefore are not found in **Initial Request Folder**.

Reviewer: ```suggestion
The RTCG dataset is continuously updated with more Clinical Genomics data from St. Jude. When you receive an email about the latest release to the dataset, you return to the Databrowser and once again select RTCG from the Studies Tab. While you request the data, rather than create a new project, you can choose to vend data to an existing project and select **RTCG Data Project** from the dropdown menu. Now you may name a new folder to vend to. We'll call this one **New Release Folder** and submit the request. Since you already have access, there is no need to complete a new DAA or collect any signatures. There will now be two folders within your **RTCG Data Project** with the **Initial Request Folder** having most of the data, while **New Release Folder** only has the new files that have been released since the first request was made. Therefore, the new files are not found in **Initial Request Folder**.
```

## Completing the Data Access Agreement

!!!info 
If you already have access to the data or are requesting open-access data, you will not be prompted to go through this section.
!!!

<!--Every person who requests access to our controlled-access data must sign the [Data Access Agreement (DAA)](../glossary/#data-access-agreement). If you are located in the United States of America, you can opt in to completing the DAA through an electronic setup wizard. If you are not located in the USA, or would like to complete the form manually, you can follow our instructions on [Filling Out The Data Access Agreement](../how-to-fill-out-daa).-->

> Through the Electronic Data Access Agreement wizard, you will add all of the necessary information needed for the [Data Access Agreement (DAA)](../glossary/#data-access-agreement). Once you have completed the setup wizard, if you are located in the United States of America, you can opt in to using Docusign to collect the required signatures - the form will be sent to you and necessary signatories through email via [DocuSign](https://www.docusign.com). If you are not located in the USA, or would like to complete the form manually, you will print the DAA and collect signatures manually, then upload and submit the request. You can learn more about [Filling Out The Data Access Agreement here](../how-to-fill-out-daa).

Reviewer: ```suggestion
Through the Electronic Data Access Agreement wizard, you will add all of the necessary information needed for the [Data Access Agreement (DAA)](../glossary/#data-access-agreement). Once you have completed the setup wizard, if you are located in the United States of America, you can opt in to using DocuSign to collect the required signatures - the form will be sent to you and necessary signatories through email via [DocuSign](https://www.docusign.com). If you are not located in the USA, or would like to complete the form manually, you will print the DAA and collect signatures manually, then upload and submit the request. You can learn more about [Filling Out The Data Access Agreement here](../how-to-fill-out-daa).
```

!!!info Planning to Download?

> If you would like to download the data to local storage, there are extra steps you'll need to follow such as [getting additional signatures](../how-to-fill-out-daa/#data-download-permission) on your data access agreement. We recommend that you work with the data in the cloud if it's feasible; the data provided by St. Jude is free, the compute charges are reasonable, and working in the cloud helps to eliminate the long, error-prone downloading process. Porting your tools to be run in the cloud is easy, as well. We recommend you follow [this guide](../../analyzing-data/creating-a-cloud-app) to get started.

Reviewer: ```suggestion
If you would like to download the data to local storage, there are extra steps you'll need to follow such as [getting additional signatures](../how-to-fill-out-daa/#data-download-permission) on your data access agreement. We recommend that you work with the data in the cloud if it's feasible; the data provided by St. Jude is free, the compute charges are reasonable, and working in the cloud helps to eliminate the long, error-prone download process. Porting your tools to be run in the cloud is easy as well. We recommend you follow [this guide](../../analyzing-data/creating-a-cloud-app) to get started.
```

!!!

> Request approval typically takes less than 2 weeks if your data access agreement is correctly and completely filled out. You will receive automated emails from notifications@stjude.cloud at the time that your request is approved. Note that an email for each of the Data Access Units requested is sent, and they are not all approved at once, but rather as soon as each committee grants approval. 

Reviewer: ```suggestion
Requesting approval typically takes less than 2 weeks if your data access agreement is correctly and completely filled out. You will receive automated emails from notifications@stjude.cloud when your request is approved. Note that an email for each of the Data Access Units requested is sent, and they are not all approved at once, but rather as soon as each committee grants approval. 
```
